tests/regression/array_gt_legacy/implementation.cpp

```diff
@@ -10,7 +10,9 @@
 
 #include <array>
 
+#ifndef CPP_BINDGEN_GT_LEGACY
```

Comment:
Probably we should simplify this now (I used a different pattern before):
I think it should just be
`#include <cpp_bindgen/export_gt_legacy.hpp>` without the defines